frontend/src/i18n/locales/features/admin-form/sidebar/workflow/en-sg.ts

```diff
@@ -0,0 +1,134 @@
+    select: 'Select a respondent',
+    fieldsToFill: 'Fields to fill',
+    clickToEdit: 'Click to edit',
+  },
+  dynamicRespondent: {
+    title: 'An email field from the form',
+    required: 'Please select a field.',
+    mustBeEmail: 'Field is not an email field',
+    select: 'Select a field',
+  },
+  conditionalRouting: {
+    title: 'Emails assigned to options in a dropdown field',
+    addEmailsToOptions: 'Add emails to options',
+    validation: {
+      noField: 'Please select a field.',
+      notDropdown: 'Field is not an dropdown field',
+    },
+    modals: {
+      deleteStep: {
+        title: 'Delete step',
+        description:
+          'Are you sure you want to delete this step? This action cannot be undone.',
+        confirm: 'Yes, delete step',
+        cancel: "No, don't delete",
+      },
+      deleteMapping: {
+        title: 'Delete CSV file',
+        description:
+          'Are you sure you want to delete this CSV file? This action cannot be undone.',
+        confirm: 'Yes, delete CSV file',
+        cancel: "No, don't delete",
+      },
+      addMapping: {
+        step1: {
+          title: 'Add emails to options',
+          download: {
+            templateCreated:
+              'We have created a CSV template with the options from the field you selected.',
+            pleaseDownload:
+              'Please download the CSV template and add the emails for each option.',
+            button: 'Download and edit CSV template',
+            howto: {
+              title: 'How to use the CSV template:',
+              option: {
+                title: 'Option',
+                explanation: 'This contains all the options from your field.',
+                notice:
+                  'Do not edit, reorder or delete anything in this column.',
+              },
+              email: {
+                title: 'Email(s)',
+                explanation:
+                  'Add the emails to send the form to for each option.',
+                notice: 'Separate multiple email(s) with a comma.',
+              },
+              imageCaption: 'How to set up your CSV',
+            },
+          },
+          nextButton: 'Next: Upload CSV file',
+        },
+        step2: {
+          title: 'Upload your completed CSV file',
+          confirm: 'Save CSV file',
+          description: {
+            prefix: 'Please ensure that your file is saved in',
+            csv: 'comma-separated values (.csv)',
+            suffix: 'format.',
+          },
+        },
+      },
+    },
+    errors: {
+      respondentType: {
+        required: 'Please select a respondent type',
+        invalid: 'The selected respondent type is invalid',
+      },
+      csv: {
+        required: 'Please upload a CSV file',
+        addEmailsBeforeSave:
+          'You must add emails to options before saving this step.',
+        mismatchedOptions:
+          'The options in your CSV file and selected field do not match.',
```

Comment:
Possible for us to import the error message here from `shared/constants/errors.ts`?

Reply:
oh, that makes sense. Let me do this.